LayoutTests/cssom/cssvalue-comparison.html

```diff
@@ -10,71 +10,88 @@
 <div id="test" contenteditable></div>
 
 <script type="text/javascript">
-setup({ "explicit_done": true });
+setup({ "explicit_done": true })
 
-function run() {
-    var styleElement = document.createElement("style");
-    styleElement.type = "text/css";
-    var styleTextNode = document.createTextNode("");
-    styleElement.appendChild(styleTextNode);
-    document.getElementsByTagName("head")[0].appendChild(styleElement);
+function run()
+{
+    var styleElement = document.createElement("style")
+    styleElement.type = "text/css"
+    var styleTextNode = document.createTextNode("")
+    styleElement.appendChild(styleTextNode)
+    document.getElementsByTagName("head")[0].appendChild(styleElement)
 
-    var div = document.getElementById("test");
-    div.focus();
+    var div = document.getElementById("test")
+    div.focus()
 
-    function runTest(propertyName, styleSheetPropertyValue, inlineStylePropertyValue, expectedResult) {
-      styleTextNode.data = "div { " + propertyName + " : " + styleSheetPropertyValue + "; }";
-      document.execCommand('insertHTML', false, "<div id=\"insertedDiv\" style=\"" + propertyName + " : " + inlineStylePropertyValue + "; \"></div>");
-      var insertedDiv = document.getElementById('insertedDiv');
-      var result = expectedResult;
-      if (propertyName in insertedDiv.style)
-          result = insertedDiv.style[propertyName] === "";
-      insertedDiv.remove();
-      return result;
+    function runTest(propertyName, styleSheetPropertyValue, inlineStylePropertyValue)
+    {
+        styleTextNode.data = "div { " + propertyName + " : " + styleSheetPropertyValue + "; }"
+        document.execCommand('insertHTML', false, "<div id=\"insertedDiv\" style=\"" + propertyName + " : " + inlineStylePropertyValue + "; \"></div>")
+        var insertedDiv = document.getElementById('insertedDiv')
+        var result = insertedDiv.style[propertyName] === ""
```

Comment:
Is this useful? `result` is overridden below.

Reply:
I will check.

Reply:
Yes, left it in by accident. Deleted now.